### PR TITLE
Init document for open weather connector

### PR DESCRIPTION
**Pull request by ndkhanh-axonivy**

### CODE REVIEW

**Comment by ndkhanh-axonivy** [comment]
Hi @andreasbalsiger I created this pull request for the documentation step.
Could you help review this README.md file, it describes the purposes and functionality for the OpenWeather connector.
Thanks!

File: open-weather-connector-product/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Open Weather Connector
+
+Harness the power of freely available weather data in your applications with the Open Weather Connector! This tool streamlines access to OpenWeatherMap's comprehensive datasets, offering a wealth of valuable information within their free plan:
```
Comment (andreasbalsiger) [inline open-weather-connector-product/README.md:3]: Excellent text, I would slightly change it to: 

Harness the power of freely available weather data in your business processes with OpenWeather! This Axon Ivy connector streamlines access to OpenWeatherMap's comprehensive datasets, offering a wealth of valuable information within their free plan:

File: open-weather-connector-product/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Open Weather Connector
```
Comment (andreasbalsiger) [inline open-weather-connector-product/README.md:1]: If I am not wrong, the correct way to write it is: OpenWeather (no space). Can you please make sure to make these changes overall?

File: open-weather-connector-product/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Harness the power of freely available weather data in your applications with the Open Weather Connector! This tool streamlines access to OpenWeatherMap's comprehensive datasets, offering a wealth of valuable information within their free plan:
+
+* **Real-time weather:** Get current conditions for any location worldwide.
+* **Extended forecasts:** Plan ahead with multi-day predictions.
+* **Air quality:** Optimize user experience based on environmental factors.
+* **Geocoding:**
+	* **Address to coordinates:** Convert physical addresses to precise latitude and longitude values.
+	* **Coordinates to address:** Obtain the corresponding address for a given latitude and longitude.
+
```
Comment (andreasbalsiger) [inline open-weather-connector-product/README.md:11]: Add:
* Supports you with an easy-to-copy demo implementation to reduce your integration effort.

File: open-weather-connector-product/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+<br>
```
Comment (andreasbalsiger) [inline open-weather-connector-product/README.md:12]: I think you can delete this <br> isn't it?

File: open-weather-connector-product/README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+## Demo
+
+Within the demo, users can access to retrieve comprehensive **5-day weather forecasts** for any global location. Follow these straightforward steps:
```
Comment (andreasbalsiger) [inline open-weather-connector-product/README.md:16]: Change it to:
Within this demo, users can access comprehensive **5-day weather forecasts** for any location worldwide. Follow these simple steps:

File: open-weather-connector-product/README.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+1. **Precise Location Specification:** Accurately identify the desired location by entering the city name and its corresponding country code. For locations within the United States, the inclusion of the state code further refines the forecast.
+2. **Initiate Search:** Simply click the **Search** button to activate the retrieval process. The connector will efficiently retrieve and display a detailed **5-day forecast** specific to your chosen location.
+
+![Demo](images/forecast-weather-demo.png)
+
+## Setup
+
+### Application id
+The use of the Open Weather Map API is not free of charge. For development purposes there is a free version with minimal API calls. To use the connector, you must select a suitable API package via the [Open Weather Map API Developer](https://openweathermap.org/api) and generate a **api key**.
```
Comment (andreasbalsiger) [inline open-weather-connector-product/README.md:26]: The OpenWeatherMap weather API is not free to use. However, there is a free version with minimal API calls for development purposes. To use the connector, select a suitable API package via the [OpenWeatherMap API Developer](https://openweathermap.org/api) and generate an **API key**.

File: open-weather-connector-product/README.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+##### How to get api key
```
Comment (andreasbalsiger) [inline open-weather-connector-product/README.md:28]: #### How to get an API key

File: open-weather-connector-product/README.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+Within the demo, users can access to retrieve comprehensive **5-day weather forecasts** for any global location. Follow these straightforward steps:
+
+1. **Precise Location Specification:** Accurately identify the desired location by entering the city name and its corresponding country code. For locations within the United States, the inclusion of the state code further refines the forecast.
+2. **Initiate Search:** Simply click the **Search** button to activate the retrieval process. The connector will efficiently retrieve and display a detailed **5-day forecast** specific to your chosen location.
+
+![Demo](images/forecast-weather-demo.png)
+
+## Setup
+
+### Application id
```
Comment (andreasbalsiger) [inline open-weather-connector-product/README.md:25]: ### Application ID

File: open-weather-connector-product/README.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+The use of the Open Weather Map API is not free of charge. For development purposes there is a free version with minimal API calls. To use the connector, you must select a suitable API package via the [Open Weather Map API Developer](https://openweathermap.org/api) and generate a **api key**.
+
+##### How to get api key
+1. Login and go to your [Open Weather Map API keys page](https://home.openweathermap.org/api_keys)
```
Comment (andreasbalsiger) [inline open-weather-connector-product/README.md:29]: 1. Login and go to your [OpenWeatherMap API keys page](https://home.openweathermap.org/api_keys)

File: open-weather-connector-product/README.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+2. Add your api key name and generate it:
```
Comment (andreasbalsiger) [inline open-weather-connector-product/README.md:30]: 2. Add your API key name and generate it:

File: open-weather-connector-product/README.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+![Register key](images/register-api-key.png)
+3. Api key now is available:
```
Comment (andreasbalsiger) [inline open-weather-connector-product/README.md:32]: 3. API key now is available:

File: open-weather-connector-product/README.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+![Register key](images/register-successful.png)
+
+After a **api key** is available, you can store it in your project in the variables.yaml as the variable "openWeatherConnector.appId":
```
Comment (andreasbalsiger) [inline open-weather-connector-product/README.md:35]: After a **API key** is available, you can store it in your project in the variables.yaml as the variable "openWeatherConnector.appId":

**Comment by ndkhanh-axonivy** [comment]
Hi @andreasbalsiger , I've just pushed a new commit that addresses the feedback you provided. Could you please review the updated README again?

Thanks for taking a look at the document!

**Review by andreasbalsiger — COMMENTED** [review]
Thanks... I can approve the changes.

**Pull request merged**